### PR TITLE
Incorporate signs on bidirectional routes into tiles

### PR DESCRIPTION
**Pull request by noblige**

### CODE REVIEW

File: lua/graph.lua [hunk 1]
```diff
@@ -1153,6 +1153,8 @@ function filter_tags_generic(kv)
   kv["tunnel"] = tunnel[kv["tunnel"]] or "false"
   kv["toll"] = toll[kv["toll"]] or "false"
   kv["destination"] = kv["destination"]
+  kv["destination:forward"] = kv["destination:forward"]
+  kv["destination:backward"] = kv["destination:backward"]
   kv["destination:ref"] = kv["destination:ref"]
```
Comment (dgearhart) [inline lua/graph.lua]: Do you need the forward and backward functionality for destination:ref, destination:ref:to, destination:street, & destination:street:to?

Comment (noblige) [inline lua/graph.lua]: these seems to be used even very infrequently. so I didn't bother.

**Comment by dnesbitt61** [comment]
We will import data over the weekend to test this change. If all looks good we will merge early next week. If all looks good we will use this as a model to add the functionality to other destination info.

**Comment by dgearhart** [comment]
Looks good - thank you @noblige

**Pull request merged**